packages/core/database/lib/query/helpers/where.js

```diff
@@ -247,14 +248,18 @@ const applyOperator = (qb, column, operator, value) => {
       break;
     }
     case '$null': {
-      if (value) {
+      if (parseBoolean(value)) {
```

Comment:
I really don't think this makes sense to do this parsing here. I think this should be part of the convertFilters function in the convert query params phase. I don't see a good reason to allow this here

Reply:
I'm not sure where you are referring to. Would you have 5 minutes to discuss about it?

Reply:
convertFiltersQueryParams in packages/core/utils/lib/convert-query-params.js. right now it's not doing anything. that's where we would do these kind of casting. to avoid having that at the db level

Reply:
The code is already handling false but not 'false'.
I think it's better that way as it's at a programmatic level. 
But we should still be able to handle 'false', 'f', 't', 'true' at a higher level instead (ideally it would be at controller/resolver) level, but in a first iteration we can handle it at the entityService level

Reply:
Yes I thought that way to and then I saw that in `packages/core/database/lib/fields.js`, we parse booleans for a field (toDB). I was wondering if we should apply the same logic for the filters parsing and the fields parsing. Does it make sense to parse for the fields bu not the filters in the package Database? I think it can make sense but I would like a confirmation

Reply:
I think I would move that out of the DB too now.